src/main/java/com/max/grpc/orders/client/rest/models/ApiFoodItem.java

```diff
@@ -0,0 +1,63 @@
+
+package com.max.grpc.orders.client.rest.models;
+
+import com.max.grpc.orders.proto.FoodItem;
+import com.max.grpc.orders.proto.FoodType;
+
+public class ApiFoodItem {
+    private String id;
+    private String title;
+    private FoodType foodType;
+    private double weight;
+    private int price;
+
+    public String getId() {
+        return id;
+    }
+
+    public void setId(String id) {
+        this.id = id;
+    }
+
+    public String getTitle() {
+        return title;
+    }
+
+    public void setTitle(String title) {
+        this.title = title;
+    }
+
+    public FoodType getFoodType() {
+        return foodType;
+    }
+
+    public void setFoodType(FoodType foodType) {
+        this.foodType = foodType;
+    }
+
+    public double getWeight() {
+        return weight;
+    }
+
+    public void setWeight(double weight) {
+        this.weight = weight;
+    }
+
+    public int getPrice() {
+        return price;
+    }
+
+    public void setPrice(int price) {
+        this.price = price;
+    }
+
+    public static ApiFoodItem from(FoodItem protoItem) {
+        var responseItem = new ApiFoodItem();

```

Comment:
Good! If you want less code can try to use  MapStract. Great lib for this

Reply:
Switched